apps/src/blockly/addons/cdoXml.js

```diff
@@ -1,4 +1,7 @@
 export default function initializeBlocklyXml(blocklyWrapper) {
+  // Clear xml namespace
+  blocklyWrapper.utils.xml.NAME_SPACE = '';
```

Comment:
just curious: why do we want to clear the namespace?

Reply:
summarizing from offline conversation: Nokogiri, which we use for xml parsing, checks the xml namespace for selectors and stuff like that. Cdo Blockly doesn't set a namespace, so if we try to use the same parsing logic for Google Blockly with the namespace set, it breaks the parsing/selecting logic. The simplest/safest thing to do here is to just remove the namespace so that we don't have to change all of our serialization/deserialization logic.